otx/algorithms/common/tasks/training_base.py

```diff
@@ -279,6 +280,9 @@ def _init_recipe_hparam(self) -> dict:
             runner=ConfigDict(max_epochs=int(params.num_iters)),
         )
 
+    def _update_stage_module(self, stage_module: str):
+        return stage_module
```

Comment:
what was that..? just return input?

Reply:
Yes, This is base function, if there are no certain stage change rule for each task(cls, det, seg) than it do not change it's stage